sdk/tracing/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/implementation/AmqpTraceUtil.java

```diff
@@ -24,7 +24,7 @@ public static Status parseStatusMessage(String statusMessage, Throwable error) {
                 ? Status.UNKNOWN.withDescription(message)
                 : Status.UNKNOWN.withDescription(error.getClass().getSimpleName());
 
-        } else if (statusMessage.equalsIgnoreCase("success")) {
+        } else if (statusMessage != null && statusMessage.equalsIgnoreCase("success")) {
```

Comment:
given the first if-block returns, could you move this to the next line down as a regular if-block instead to make it slightly more clear.  Similarly, do you need to handle the statusMessage==null case separately so it doesn't also flow down to the Status.UNKNOWN.withDescription below?